### ISSUE TITLE
How can I disable Ctrl-F1 hotkeys in copyq

### ISSUE THREAD

**Issue by amosbird** [hluk/CopyQ#744]
In fact, is there a way to disable all global hotkeys in copyq?

**Comment by hluk** [comment]
You can disable all commands with assigned hotkeys if you open Action dialog (F5 shortcut) and run following.

```js
copyq:
cmds = commands()
for (i in cmds) {
  c = cmds[i]
  if (c.globalShortcuts != "" && c.enable) {
    popup("Disabling " + c.name + ": " + c.globalShortcuts)
    c.enable = false;
  }
}
setCommands(cmds)
```

**Comment by amosbird** [comment]
um, I did this 

![](https://la.wentropy.com/MgM8.png) 

but pressing apply or ok doesn't disable all the hotkeys. Ctrl-F1 still shows current window's title.

**Comment by hluk** [comment]
Oops, looks like bug. As workaround, you need to go to Commands dialog (F6) and just press OK to apply the command settings.

**Comment by hluk** [comment]
The bug should be fixed now in 36455c5174734ddc847b3e4f96c6de8c6ab89efe.

**Comment by amosbird** [comment]
thanks!
